cmake/FindCMSIS.cmake

```diff
@@ -72,21 +77,27 @@ function(cmsis_generate_default_linker_script FAMILY DEVICE CORE)
     stm32_get_memory_info(FAMILY ${FAMILY} DEVICE ${DEVICE} CORE ${CORE} HEAP SIZE HEAP_SIZE)
     stm32_get_memory_info(FAMILY ${FAMILY} DEVICE ${DEVICE} CORE ${CORE} STACK SIZE STACK_SIZE)
```

Comment:
this should all be included in the `if MP1` as you won't use the information for creating the ld script

Reply:
Done